lib/users/model.js

```diff
@@ -36,13 +36,14 @@ const structure = {
       type: 'email',
       id: 'email',
       name: 'Email',
+      required: 'save',
       description: 'Email address for your account',
     },
     {
       type: 'password',
       id: 'password',
       name: 'Password',
-      description: 'Gotta get in',
+      description: 'CAUTION: Adding a password here will change the user\'s password.',
```

Comment:
No, that's fine. Just trying to do code review based on the provided story. This code doesn't fully resolve the story it's for, but I'll merge it given that there is another story in progress that will encompass the required changes